src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/cli/usecases/ComposeEmailCLIPrompt.java

```diff
@@ -34,9 +38,43 @@ public void start() {
       return;
     }
     askTextContent();
+    askScheduledSend();
     sendEmail();
   }
 
+  private void askScheduledSend() {
+    DateTimeFormatter formatter =
+        DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").withZone(ZoneId.systemDefault());
+    ZonedDateTime scheduledDate = ZonedDateTime.now();
+
```

Comment:
For all of this format parsing, we should use a service. Its too much logic for the presentation layer